pkg/controller/import-controller.go

```diff
@@ -625,9 +675,7 @@ func makeImporterPodSpec(namespace, image, verbose, pullPolicy string, podEnvVar
 			Labels: map[string]string{
 				common.CDILabelKey:       common.CDILabelValue,
 				common.CDIComponentLabel: common.ImporterPodName,
-				// this label is used when searching for a pvc's import pod.
-				LabelImportPvc:         pvc.Name,
```

Comment:
can `LabelImportPvc` const be deleted?

Reply:
all functional tests pass, no usage fund so far,
do you have any information where it might be needed,